src/pages/Account/Tabs/ModulesTab/Tabs.tsx

```diff
@@ -39,13 +39,13 @@ function TabPanel({value, address}: TabPanelProps): JSX.Element {
 function ModulesTabs({address}: {address: string}): JSX.Element {
   const theme = useTheme();
   const tabValues = Object.keys(TabComponents) as TabValue[];
-  const {modulesTab} = useParams();
+  const {selectedModuleName, modulesTab} = useParams();
   const navigate = useNavigate();
   const value =
     modulesTab === undefined ? tabValues[0] : (modulesTab as TabValue);
 
   const handleChange = (event: React.SyntheticEvent, newValue: TabValue) => {
-    navigate(`/account/${address}/modules/${newValue}`);
+    navigate(`/account/${address}/modules/${selectedModuleName}/${newValue}`);
```

Comment:
now clicking btw module names actually refresh the page, instead of just update the source code & abi on the right side. when we switch tabs in the account page, it does change url param but page's not refreshed. lets get it fixed!

Reply:
not sure I quite understand? I don't think it does a full page refresh - it visually looks all the same as before adding the deeplinking and it definately doesn't do a full page reload and reloads the JS assets.

If you can take a video of what you're seeing that might be helpful but if its easy to explain that's also fine.

Reply:
lol i was playing around with something else in the dev env that mess with this `<Link>`. i can confirm it's all good, no page reload!